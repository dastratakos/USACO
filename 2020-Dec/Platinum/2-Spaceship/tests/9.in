60 58 60
011011100111110111100111111111000111110111111100001110011101
100101101110101011011000101100001011111111101011100100011111
111110111001011011001111111111111010101111101010000011101111
011101010101101100011111111011001011011101111110010010111111
001111010001110101001011101111111011101110011100111011110110
011111110000111111111111011111101110101110001101110110111111
110110100111001101111111010111001011110110000000111010111111
101110010111011101100111001011111111011001000110111101110110
011111111011000010111111101111100000000111001001111111101111
110100111011110100100001111001101111100111110101001101011001
001110000001100101100111111001110011100111001111001010110110
111100101100001000000111101001001111010001101111011111010100
010111010111011011111010000111110101001111111111110001111111
110110000011011011111110111110101001111000111110110011110011
011111110100101111110111101101011011010101001111101111111111
111011110111111011111110000100110000101110111101010001101001
011010111101100111111100101010111011100111111011100001101110
110000111001110101101011100110110100110111110111000001001110
010011101100101110110001001111111000100110100101011110010111
011111011100111111111101111101110000010011111111101010111110
111110111011010011101011100110011011101001001111010001111110
011111011111110100111111100001011111111111110001111101110110
101101111100101100111111011101111110110111011101100111100110
001110110000101100110111111101010111111111111110100101111111
110010111110110011001001001101110101101011111010111111011111
100111011000110111111011111001101110011101010001101101001010
110010111011111101000100101110101011000111111111111010101100
111110101101001111110100010111010001111100010110010101000010
101100111101110101110111100111111100011101101110111011111001
101010111111111111101100011101100110111011111001111110110111
011001000110111110111001110101111011001111111001101100010010
111111111101110011100011110111010110110111100011011110111000
100010110001110110001111001110110001111110111111111111111110
001101100110101111000010110100101011101011100111110101010110
111011001101110111011011111111111101101110111111111101111110
101011101100100001111111001001101111011010111101011011111110
101111111010111110111101001001011111111110111111110111011111
010111111101110111111110011011011111001101001000010111100110
100011101011011100110111001101111001111101100001110111100011
011111111111110111101110111100111001101101011111111111111000
101101111110111100100110011100010000110000001110101110110111
110011100111101111111010001111000011011100011111011100011111
101110111111001101110001111100111100111101001110110111011111
011011110011100001101011100111101101001101101001111101110101
010011010010100011111011111101111011100111011111101110110000
111111111100000011011000001010110011111101001101011101101101
111010111100101111101011011101100110001111111000100011100110
110110101101111011011010001111111110011101111111011111011110
111110111011111111111111101011111001010100111110101111001111
101001111111101111010000101110011001101101111101110011011011
111010101011010110110100101111011111011101111111111011111111
111111101101101001110010111101001100011111011110110111011001
111100001110011001111011111101111110111011000101111110111110
111110101001111001001010011011010101011100101111111110001111
010001010110110011101011011111111011001011100110000111001111
110011111011111101111110110111110011011110111101011100100011
010111101010010111011100111111111011110011001111111000011111
011000111100111100101000100111011111111100011111110101110000
011011101100101111100110101010001110010111001101110011110111
001111001101101011101101111111101111110100010110111011011111
57 19 58 38
57 13 58 48
57 43 58 21
57 26 58 32
57 58 58 8
57 47 58 3
57 60 58 19
57 44 58 21
57 18 58 58
57 46 58 27
57 30 58 41
57 35 58 54
57 23 58 46
57 47 58 15
57 51 58 27
57 19 58 19
57 45 58 4
57 34 58 35
57 43 58 12
57 46 58 23
57 21 58 38
57 20 58 36
57 33 58 49
57 37 58 47
57 49 58 4
57 58 58 1
57 28 58 19
57 4 58 52
57 33 58 53
57 40 58 13
57 22 58 8
57 39 58 15
57 8 58 23
57 4 58 14
57 3 58 10
57 60 58 51
57 42 58 18
57 44 58 57
57 15 58 37
57 23 58 53
57 48 58 41
57 23 58 27
57 46 58 49
57 30 58 53
57 56 58 57
57 28 58 44
57 12 58 48
57 23 58 7
57 16 58 56
57 33 58 29
57 16 58 9
57 36 58 60
57 52 58 22
57 33 58 47
57 21 58 55
57 41 58 48
57 59 58 36
57 38 58 22
57 22 58 48
57 2 58 36
