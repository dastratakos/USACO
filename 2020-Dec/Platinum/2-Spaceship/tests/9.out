536893136
77755822
704219333
132300389
312202386
976315275
435584534
636537142
771953265
295369137
205715958
197733966
974812356
729205407
166357405
883531790
41360763
983246967
885267629
600141328
31236295
334428405
127411098
27972521
336161355
716194181
626330687
885046964
129722852
682895579
493296084
493405422
760098580
86622326
600420769
595267159
716353980
270148858
316210593
474141956
705547173
6477409
649916812
271006055
93646419
143163274
486776782
59286907
880186476
694901507
384682018
812014291
903448678
807399254
592921668
421216380
972087399
860724049
544185029
608195241
